internal/rbd/driver/driver.go

```diff
@@ -43,6 +43,8 @@ type Driver struct {
 	cas *csiaddons.CSIAddonsServer
 }
 
+var nodeLabels, topology, crushLocationMap map[string]string
```

Comment:
Let's just add volumeLocks to the rs controller and node server structs directly instead of inheriting and adding global variables.